pkg/sql/sqlbase/structured.proto

```diff
@@ -1002,17 +1002,13 @@ message TypeDescriptor {
   // id is the globally unique ID for this type.
   optional uint32 id = 4 [(gogoproto.nullable) = false, (gogoproto.customname) = "ID", (gogoproto.casttype) = "ID"];
 
-  // oid is the Postgres compatible OID for this type. It is intended to be
-  // used only within catalog tables.
-  optional uint32 oid = 5 [(gogoproto.nullable) = false, (gogoproto.customname) = "OID"];
-
   // Represents the kind of type that this type descriptor represents.
   enum Kind {
     // Represents a user defined enum type.
     ENUM = 0;
     // Add more entries as we support more user defined types.
   }
-  optional Kind kind = 6 [(gogoproto.nullable) = false];
+  optional Kind kind = 5 [(gogoproto.nullable) = false];
```

Comment:
Careful - you can't do stuff like this! Changing the ids of protos is a big nono - it'll break the representation on disk. If you want to delete an old one, and again I'm not sure why you would want to, you'd reserve the field. The numbers are important - that's what defines the serialization format.

Reply:
These aren't persisted yet -- no cluster will have these type descriptors stored on disk, so I thought that this is reasonable to do.

Reply:
Oh, my bad.